test/features/environment.py

```diff
@@ -139,6 +142,48 @@ def before_scenario(context, scenario):
     # Set personas.
     context.personas = PERSONAS
 
+    for tag in scenario.tags:
+        FIXTURE_NAME = 0
+        PARAMS = slice(1, None)
+
+        parts = tag.split("::")
+
+        if parts[FIXTURE_NAME].startswith("fixture.dataset_with_schema"):
+            use_fixture(dataset_with_schema, context, parts[PARAMS])
+        elif parts[FIXTURE_NAME].startswith("fixture.create_resource_for_dataset_with_params"):
+            use_fixture(create_resource_for_dataset_with_params, context, parts[PARAMS])
```

Comment:
comments please, what are you trying to do with tags which can't be done inside the bdd steps or examples?